Platforms/QemuSbsaPkg/Binaries/secureboot_binaries_ext_dep.yaml

```diff
@@ -0,0 +1,15 @@
+##
+# Download the secureboot binary blobs for the default Db, 3PDb, Dbx, KEK, and Pk
+##

```

Comment:
Please go ahead and add the Microsoft copyright & BSD SPDX ID if there's going to be a file header.

Reply:
Added. Thanks!